src/abi_impl.rs

```diff
@@ -140,7 +166,7 @@ pub(crate) fn assembly_script_call_module(
     let address = &get_string(memory, address)?;
     let function = &get_string(memory, function)?;
     let param = &get_string(memory, param)?;
-    let response = call_module(env, address, function, param)?;
+    let response = call_module(env, address, function, param, 0)?;
```

Comment:
this  assembly_script_call_module function should also take a coins parameter and pass it to call_module, otherwise it won't be usable from smart contracts

Reply:
and massa.ts should be updated to reflect this new parameter in the call function